docs/en/02_Developer_Guides/00_Model/06_SearchFilters.md

```diff
@@ -6,7 +6,7 @@ icon: search
 
 # SearchFilter Modifiers
 
-The `filter` and `exclude` operations specify exact matches by default. However, there are a number of suffixes that
+The `filter` and `exclude` operations specify exact matches by default. However, for `DataList` there are a number of suffixes that
```

Comment:
```suggestion
The `filter` and `exclude` operations specify exact matches by default. However, when filtering `DataList`s, there are a number of suffixes that
```

Reply:
Thanks @Cheddam, PR is updated with your suggested change. (I did amend the previous commit)